index.bs

```diff
@@ -112,32 +110,32 @@ if (screen.isExtended) {
 ### Requesting extended screen information ### {#usage-overview-screen-information}
 <!-- ====================================================================== -->
 
-After determining that there are multiple screens, information can be requested about them using the {{Window/getScreens()}} method. This method may prompt the user for permission. The resulting object lets developers enumerate screens, inspect screen properties, and watch for changes.
+After determining that there are multiple screens, information can be requested about them using the {{Window/getScreenDetails()}} method. This method may prompt the user for permission. The resulting object lets developers enumerate screens, inspect screen properties, and watch for changes.
 
 ```js
 try {
-  const screensInfo = await window.getScreens();
+  const screenDetails = await window.getScreenDetails();
 
   // Do something with the screens right now.
-  processScreens(screensInfo);
+  processScreenDetails(screenDetails);
 
   // And also if the screens change in the future.
-  screensInfo.onscreenschange = e => {
-    processScreens(screensInfo);
+  screenDetails.onscreenschange = e => {
+    processScreenDetails(screenDetails);
   };
 } catch (ex) {
   /* permission denied, or other error */
 }
 
-function processScreens(screensInfo) {
+function processScreenDetails(screenDetails) {
   // Build a UI listing screens.
   clearScreenList();
-  screensInfo.screens.forEach(screen => {
+  screenDetails.screens.forEach(screen => {
     addToScreenList({id: screen.id, name: screen.label, screen: screen});
   });
 
   // Set the selection to the current screen.
-  selectCurrentInScreenList(screensInfo.currentScreen.id);
+  selectCurrentInScreenList(screenDetails.currentScreen.id);
```

Comment:
Here as well.

Reply:
Done.  (Just passed the currentScreen directly.)